packages/legacy/tools/src/models/package/package.constants.ts

```diff
@@ -11,9 +11,9 @@ const PATTERNS = {
  * Test directories for organizing test runners.
  */
 const TEST_DIRECTORIES = {
-  INTEGRATION: './integration',
```

Comment:
code looks good , need the intergation and coverage to be fixed

Reply:
These tests are fixed now.